be/src/olap/rowset/segment_v2/inverted_index_reader.cpp

```diff
@@ -607,6 +639,15 @@ Status InvertedIndexIterator::try_read_from_inverted_index(const std::string& co
                                                            const void* query_value,
                                                            InvertedIndexQueryType query_type,
                                                            uint32_t* count) {
+    // NOTE: only bkd index support try read now.
+    if (query_type == InvertedIndexQueryType::GREATER_EQUAL_QUERY ||
```

Comment:
may be change to other than match?

Reply:
yes, but i feel it's not necessary, match query type also has several types: MATCH_ANY_QUERY, MATCH_ALL_QUERY, MATCH_PHRASE_QUERY